res/controllers/Traktor-Kontrol-S4-MK2-hid-scripts.js

```diff
@@ -0,0 +1,1275 @@
+/****************************************************************/
+/*      Traktor Kontrol S4 MK2 HID controller script v1.00      */
+/*      Copyright (C) 2015, the Mixxx Team                      */
+/*      but feel free to tweak this to your heart's content!    */
+/*      For Mixxx version 2.0                                   */
+/****************************************************************/
+
+
+// TODO:
+// * Loop size readout / selection
+// * Find a use for snap / master / other unused buttons?
+
+TraktorS4MK2 = new function() {
```

Comment:
You can make the configuration flags standalone variables at the top and then copy them in. 

```javascript
var RemixButtonSlotAction = "Samples";

var TraktorS4MK2 = new function() { };

TraktorS4MK2. RemixButtonSlotAction = RemixButtonSlotAction;
```